examples/aws_lambda/aws_lambda.py

```diff
@@ -1,7 +1,5 @@
 import logging
-import sys
 
-sys.path.insert(1, "vendor")
```

Comment:
It doesn't look like this works. [AWS Chalice seems to include `vendor/` automatically](https://aws.github.io/chalice/topics/packaging.html) but AWS Lambda not so much. I can't find more information on this, but I do remember before this did work! But, it has been probably 2 years or so since I've used Python in Lambda, so perhaps that behaviour has changed.

Reply:
Good catch 👍